typescript/codepipeline-using-cross-account-repos/index.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import cdk = require('@aws-cdk/core');
+import codecommit= require('@aws-cdk/aws-codecommit');
+import codepipeline = require('@aws-cdk/aws-codepipeline');
+import codepipeline_actions = require('@aws-cdk/aws-codepipeline-actions');
+import iam = require('@aws-cdk/aws-iam');
+import s3 = require('@aws-cdk/aws-s3');
+import kms = require('@aws-cdk/aws-kms');
+ 
+const app = new cdk.App();
+const devEnv = {region: 'us-west-2', account: '123123123123'} 
```

Comment:
Example account numbers should be of the format 11111111111, 22222222222, etc.....

There are a few account number spread throughout this example and it’s not clear which ones a re dummies and which ones are artifacts from your building/testing.

Reply:
Woops, I accidentally pushed my account id, good call